jetstream/tests/core/test_server.py

```diff
@@ -60,22 +61,26 @@ async def test_server(
     """Sets up a server and requests token responses."""
     ######################### Server side ######################################
     port = portpicker.pick_unused_port()
+    metrics_port = portpicker.pick_unused_port()
```

Comment:
Could you create a separate test to test prometheus metric enabled? 1 test for not enabled, 1 test for enabled.